gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/ApiMetadata.java

```diff
@@ -36,4 +35,118 @@ public class ApiMetadata {
     String value;
 
     String defaultValue;
+
```

Comment:
why not using lombok here ? (getters setters equals hashcode and toString)

Reply:
I mess-up I think I de-lombokize the class for a test but I forgot to revert back 🙈